src/lib/HeadHrefLangs.svelte

```diff
@@ -0,0 +1,9 @@
+<script lang="ts">
+	import { locales } from '$i18n/i18n-util'
+	import { page } from '$app/stores'
+	import { replaceLocaleInUrl } from '../utils'
+</script>
+
+{#each locales as l}
+	<link rel="alternate" hreflang={l} href={`${replaceLocaleInUrl($page.url, l, true)}`} />
```

Comment:
Should the currently active language really be added as an `alternate` link?

Reply:
Yes, it is mandatory to include the current page, as per the [documentation](https://developers.google.com/search/docs/specialty/international/localized-versions#html):

> For each variation of the page, include a set of <link> elements in the <head> element, one link for each page variant **including itself**.

Reply:
Right 👍
Can you please also add [x-default](https://developers.google.com/search/docs/specialty/international/localized-versions#xdefault)?

Reply:
I considered it but was not sure which url to use for it. Since the / url is a permanent redirect, I'm not sure we can add it as x-default.

Reply:
You can generate the link dependeing on the `baseLocale`, exported from `i18n-util.ts`.

Reply:
Alright I'll try to do it tomorrow if I find some time.